### PR TITLE
rebar3 3.0.0-beta.3

### PR DESCRIPTION
**Pull request by niku**
https://github.com/rebar/rebar3/releases/tag/3.0.0-beta.3

### CODE REVIEW

**Comment by dunn** [comment]
Failing to build on all platforms: http://bot.brew.sh/job/Homebrew%20Devel-Only%20Pull%20Requests/99/

Did it work for you locally?

**Comment by niku** [comment]
Ah, sorry.
I checked working fine on local environment, and pushed again.

**Comment by niku** [comment]
I fixed errors.
http://bot.brew.sh/job/Homebrew%20Devel-Only%20Pull%20Requests/116/

Would you merge it?

File: Formula/rebar3.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Rebar3 < Formula
   homepage "http://www.rebar3.org"
 
   devel do
-    url "https://github.com/rebar/rebar3/archive/beta-2.tar.gz"
-    version "3.0.0-beta-2"
-    sha256 "0aa014290f5d971206fc79f48d0e3047266e09a6124f044b7e6f7a396b10d831"
+    # We cannot use tar/zip file.
+    # It needs git repository for versioning at bootstrap.
+    url "https://github.com/rebar/rebar3.git", :revision => "3.0.0-beta.3"
```
Comment (dunn) [inline Formula/rebar3.rb]: `3.0.0-beta.3` is the tag, so this should be
```ruby
url "https://github.com/rebar/rebar3.git",
  :tag => "3.0.0-beta.3", 
  :revision => "4177c75a7c709b615962b1a885c119a51ed0aab0"

**Comment by niku** [comment]
Thank you for comment :smiley: 
I adopt your comment into the code, and pushed it.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-devel-only/commit/51fbdadaaaa55e2312aec9c172a589f620d2022a. Thank you for your contribution to Homebrew @niku! :smiley_cat:

**Comment by niku** [comment]
Thank you! :heart:

**Pull request closed without merging**